LiteCore/Support/EncryptedStream.cc

```diff
@@ -19,7 +19,7 @@
 #include "SecureRandomize.hh"
 #include "SecureSymmetricCrypto.hh"
 #include "Endian.hh"
-
+#include <minmax.h>
```

Comment:
What is this header from? Xcode doesn't recognize it. Curious why adding this header is required, if there are no source changes in this file.